enhancements/20220525-system-logging.md

```diff
@@ -0,0 +1,65 @@
+# System Logging
+
+## Summary
+
+We need to be able to support exporting harvester system logs outside the cluster, to allow for collecting and analyzing those logs to aid in debugging and harvester management. 
+
+### Related Issues
+
+- https://github.com/harvester/harvester/issues/577
+
+## Motivation
+
+### Goals
+
+List the specific goals of the enhancement. How will we know that this has succeeded?
+
+- The user can aggregate harvester logs in a centralized location
+- The user can view the aggregated harvester logs outside the cluster
+
+### Non-goals
+
+- this enhancement does not cover integration into the harvester / rancher UI (but ideally this will eventually be implemented as well)
+
+## Proposal
+
+We will deploy a new `ManagedChart` in the [Harvester Installer](https://github.com/harvester/harvester-installer) to
+install. The `ManagedChart` will deploy a `ClusterFlow` to select and aggregate interesting logs, which can be managed
+by a new `ClusterOutput` crd. The `ClusterOutput` can then be configured by settings from the harvester UI.
+
+### User Stories
+
+#### Easily View Harvester Logs
```

Comment:
Please add more stories of:
(1) How logging feature is achieved, it looks to be available after installation
(2) How to turn on/turn off logging; how to set wanted log levels
(3) How to config remote server, how many servers are allowed in parallel
(4) Do we support `push` or `pull` of logging, or both ?
(5) In case the remote server is not connectable, how long will the log be saved locally
(6) Will there be a local centralized log center in Harvester cluster to store the aggregated log ?
(7) For local UI view of log, what kind of filter will be supported ?